VaticanBAV/et/BAVet119.xml

```diff
@@ -0,0 +1,552 @@
+    <teiHeader xml:base="https://betamasaheft.eu/">
+      <fileDesc>
+          <titleStmt>
+              <title xml:lang="gez" xml:id="t1">ዐውደ፡ ነገሥት፡</title>
+              <title xml:lang="gez" corresp="#t1" type="normalized">ʿAwda nagaśt</title>
+              <title xml:lang="en" corresp="#t1">Cycle of the kings and other treatises and tables on divination and astrology</title>
+              <editor key="MV"/>
+              <editor key="AB" role="generalEditor"/>
+              <funder>Akademie der Wissenschaften in Hamburg</funder>
+          </titleStmt>
+          <publicationStmt>
+              <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+              <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale Forschungsumgebung / Beta maṣāḥǝft</publisher>
+              <pubPlace>Hamburg</pubPlace>
+              <availability>
+                  <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                      licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+              </availability>
+          </publicationStmt>
+          <sourceDesc>
+              <msDesc xml:id="ms">
+                  <msIdentifier>
+                      <repository ref="INS0003BAV"/>
+                      <collection>Aethiopici</collection>
+                      <idno>Aeth.119</idno>
+                  </msIdentifier>
+                 
+                  <msContents>
+                      <summary/>
+                      <msItem xml:id="ms_i1">
+                          <locus from="3r" to="36v"/>
+                          <title type="complete" ref="LIT1171Awdana">ʿAwda nagaśt</title>
+                          <msItem xml:id="ms_i1.1">
+                              <locus target="#3r"/>
+                              <title type="complete">List of names and letters with which portions of the circle are annotated.</title>
+                          </msItem>
+                          <msItem xml:id="ms_i1.2">
+                              <locus target="#3v"/>
+                              <title>Figure consisting of a cross with four human faces at the ends 
+                                  of the four arms and an eye in the middle of each arm. 
+                                  Four circles, divided into four parts, fill the space between the arms.
+                                  Future events are indicated with this figure.</title>
+                          </msItem>
+                          <msItem xml:id="ms_i1.3">
+                              <locus from="4r" to="11v"/>
+                              <title type="complete" ref="LIT1171Awdana#tables">Sixteen circles, each divided into sixteen parts.</title>
+                              <incipit xml:lang="gez">
+                                  ፩ ዓውደ፡ ሆይ። ሀ ፩ትረክቦ፡ ለዜና፡ መፍቅድ፡ በባሕረ፡ ፃና።
```

Comment:
Unless there are no word dividers, I would prefer to have no spaces between letters and numbers:
፩ዓውደ፡ ሆይ። ሀ፩ትረክቦ፡